src/drivers/nuopc/ice_comp_nuopc.F90

```diff
@@ -89,6 +89,7 @@ module ice_comp_nuopc
   real(R8)               :: orb_obliq       ! attribute - obliquity in degrees
   real(R8)               :: orb_mvelp       ! attribute - moving vernal equinox longitude
   real(R8)               :: orb_eccen       ! attribute and update-  orbital eccentricity
+  logical                :: masterproc
```

Comment:
I had added mastertask as a logical. Should this be removed here?

Reply:
I am closing  this PR in terms of a new one that utilizes CDEPS for prescribed ice and removes all references to MCT from the nuopc directory.